CONTRIBUTING.md

```
@@ -0,0 +1,124 @@
+```
+
+The project provides automated style, tests and coverage checks:
+
+``` shell
+$ make check
+```
+
+You can run them separately:
+
+``` shell
+$ make pre-commit
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)